### PR TITLE
UI for info bar

### PR DESCRIPTION
**Pull request by samuelli**
image

### CODE REVIEW

File: client/src/catalog-element.html [hunk 1]
```diff
@@ -12,6 +12,10 @@
 <dom-module id="catalog-element">
   <template>
     <style>
+      * {
```
Comment (dstockwell) [inline client/src/catalog-element.html]: Does this need to match everything?

File: client/src/catalog-element.html [hunk 2]
```diff
@@ -166,15 +214,34 @@ <h1 id="repo-title">[[repo]]</h1>
...
+          <h3>Dependencies</h3>
+          <template is="dom-repeat" items="[[_dependencies()]]">
+            <div>[[item]]</div>
+          </template>
+        </div>
+        <div id="info-graph">
+          <h3>Latest commit [[_timeSinceString(data.updated_at)]] ago</h3>
+        </div>
+        <div id="info-contributors">
+          <h3>[[data.contributors.length]] contributors</h3>
```
Comment (dstockwell) [inline client/src/catalog-element.html]: But what if there's only one :P

File: client/src/catalog-element.html (hunk 1 above)
Comment (samuelli) [inline client/src/catalog-element.html]: No, but its a pretty common pattern.

File: client/src/catalog-element.html (hunk 2 above)
Comment (samuelli) [inline client/src/catalog-element.html]: TODO

Comment (samuelli) [inline client/src/catalog-element.html]: I'll address this when it moves to an element

**Pull request merged**